src/broadcast/types.ts

```diff
@@ -14,3 +16,46 @@ export interface StartBroadcastConfig {
   authToken: string;
   name?: string;
 }
+
+export enum BroadcastEvent {
+  slippiStatusChange = "slippiStatusChange",
+  dolphinStatusChange = "dolphinStatusChange",
+  error = "broadcastError",
+  LOG = "log",
```

Comment:
The log event doesn't match the naming convention of the other events. We should probably coerce the others to be in upper snake case or leave it for another PR.

Reply:
whoops, didn't notice the difference when I copied it from SpectateEvents.